linode/manager
# feat: [M3-7642] – Support IPv4 Ranges data in "Unassign Linodes" drawer

## Description 📝
(opened in lieu of https://github.com/linode/manager/pull/10111 being closed due to git history issues)

> [!NOTE] 
> This branches off from https://github.com/linode/manager/pull/10089 -- the diff should be reduced down to two files (`SubnetUnassignLinodesDrawer.tsx` and the changeset) once that PR is merged into the release branch (`staging-off-release`).

Support IPv4 Ranges data in "Unassign Linodes" drawer.

## Changes  🔄
- Use `RemovableSelectionsListTable` in Unassign drawer to display VPC IPv4 and VPC IPv4 Ranges data

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-01-26 at 12 03 01 PM](https://github.com/linode/manager/assets/114682940/1e8b016c-f56a-4d10-9e3c-7e0b56245725) | ![Screenshot 2024-01-26 at 12 02 38 PM](https://github.com/linode/manager/assets/114682940/cde2449d-0719-4e0e-92ed-43a25e1e15af) |

## How to test 🧪
### Verification steps 
Have the `vpc-extra-beta` tag on your account and a VPC containing at least one subnet with a linode assigned to it. Confirm that:

- You see a table in the "Linodes to be Unassigned from Subnet" section
- When you select a linode, the table is populated with its data
  - If you select one with no IPv4 Ranges, you should see "None" in that cell
 - Clicking the "X" for that table row removes it (and clicking "X" in the LinodeSelect should clear all selections from the table)

## As an Author I have considered 🤔
